lib/CustomFields/pages/ViewCustomFieldsSettings/ViewCustomFieldsSettings.js

```diff
@@ -67,7 +64,7 @@ const ViewCustomFieldsSettings = ({
 
   const { calloutRef } = useLoadingErrorCallout(customFieldsFetchFailed || sectionTitleFetchFailed);
 
-  const customFieldsExist = customFieldsLoaded && !isEmpty(customFields);
+  const isNewCustomField = isEmpty(customFields) && !!sectionTitle.value;
```

Comment:
what about renaming it to something like `noDataSaved`? it guess I would make more sense since the name `isNewCustomField` doesn't imply that the variable value is somehow dependent on the `sectionTitle`

Reply:
Nice suggestion, thanks!